tests/integrations/raftstore/test_lease_read.rs

```diff
@@ -403,3 +403,29 @@ fn test_read_index_when_transfer_leader() {
     );
     drop(cluster);
 }
+
+#[test]
+fn test_local_read_cache() {
```

Comment:
I think the test flow is:
1) peer become leader and trigger a read
2) remove peer
3) add this peer back again and became leader again
4) trigger read

Reply:
it does so.